sql/src/main/scala/org/apache/spark/sql/sedona_sql/expressions/st_functions.scala

```diff
@@ -75,8 +74,8 @@ object st_functions extends DataFrameAPI {
   def ST_Collect(geoms: String): Column = wrapExpression[ST_Collect](geoms)
   def ST_Collect(geoms: Any*): Column = wrapVarArgExpression[ST_Collect](geoms)
 
-  def ST_CollectionExtract(collection: Column): Column = wrapExpression[ST_CollectionExtract](collection)
-  def ST_CollectionExtract(collection: String): Column = wrapExpression[ST_CollectionExtract](collection)
+  def ST_CollectionExtract(collection: Column): Column = wrapExpression[ST_CollectionExtract](collection, null)
```

Comment:
If the API gets affected, maybe update the API doc as well?

Reply:
it won't affect API, we just change the way to handle such overloading like expressions